src/library_webgl2.js

```diff
@@ -723,21 +723,41 @@ var LibraryWebGL2 = {
     GL.syncs[id] = null;
   },
 
+#if WASM_BIGINT
+  glClientWaitSync__sig: 'iiii',
+#else
   glClientWaitSync__sig: 'iiiii',
   glClientWaitSync__deps: ['$convertI32PairToI53'],
```

Comment:
You can do `glClientWaitSync__deps: i53ConversionDeps,` and avoid the if/else block.

Reply:
Unfortunately that will add a dependency to `bigintToI53Checked` or `convertI32PairToI53Checked`, but in this case the function `convertI32PairToI53` is expected.